pkg/detector/ospkg/debian/debian.go

```diff
@@ -102,8 +102,7 @@ func (s *Scanner) Detect(osVer string, pkgs []ftypes.Package) ([]types.DetectedV
 				VulnerabilityID:  adv.VulnerabilityID,
 				PkgName:          pkg.Name,
 				InstalledVersion: installed,
-				//FixedVersion: adv.FixedVersion, // TODO: Why is this missing?
```

Comment:
Would you know why is this missing?

Reply:
This is because this bucket stores only unfixed vulnerabilities. I'll add a comment.